packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1115,6 +1115,7 @@ function detachFiber(current: Fiber) {
   current.pendingProps = null;
   current.memoizedProps = null;
   current.stateNode = null;
+  current.alternate = null;
   if (alternate !== null) {
     detachFiber(alternate);
```

Comment:
You'd need to move the null out after this conditional.

Reply:
Heh. I just noticed that this will call current -> alternate -> current again.

Maybe we should set:
```js
alternate. alternate = null;
```
...before calling?